.github/CHANGELOG.md

```diff
@@ -19,7 +21,7 @@
   n_wires = 20
   n_layers = 2
   
-  dev = qml.device('lightning.gpu', wires= n_wires, mpi=True, mpi_buf_size=1)
+  dev = qml.device('lightning.gpu', wires= n_wires, mpi=True)
```

Comment:
Can we use 4 spaces for indenting the Python code here?

Reply:
Done!